basic-examples/serial-data_passing/README.md

```diff
@@ -11,51 +11,60 @@ case using the template plugin) that takes its `name` input from the `fqdn` outp
 returned by the metadata plugin. All steps are run via the default deployer (defined in
 `config.yaml`), and their success outputs are reported.
```

Comment:
You might want to do a global search for `config.yaml` and make sure that it only shows up in the few places where it should (I believe that this is not one of them).